lib/open_api/serializer.rb

```diff
@@ -5,7 +5,7 @@ def as_json(arg = {})
       encrypted_columns_set = (self.class.try(:encrypted_columns) || []).to_set
       encryption_filtered = previous.except(*encrypted_columns_set)
       return encryption_filtered unless arg.key?(:prefixes)
-      /\/v(?<major>\d+)[x\.]?(?<minor>\d+)?\// =~ arg[:prefixes].first
+      /\A\/?(?<api>\w+)\/v(?<major>\d+)[x\.]?(?<minor>\d+)?\// =~ arg[:prefixes].first
```

Comment:
Not sure I understand what this change is doing. Seems like the old expression would match both `/internal/v0.0/authentications/1` and `/api/v0.0/authentications/1`

Reply:
The old regex did not allow for a leading `/`.  So, `/api/v0.0/authentications/1` would not be parsed properly, but `api/v0.0/authentications/1` would.

Reply:
Got it, thanks 👍

Reply:
Will this break if we add additional path components prior to the version? For example `/r/insights/platform/topological-inventory/v0.0/...`

I feel like the multiple path components won't match the new `\w+` for the `api` capture group.

Reply:
Fixed and added tests.